CollAction.Tests/ImageServicesTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿using CollAction.Services.Image;
+using Hangfire;
+using Hangfire.Common;
+using Hangfire.States;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using System;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace CollAction.Tests
+{
+    [TestClass]
+    public sealed class ImageServicesTests
```

Comment:
Could may be name tests to to show it's integration tests and not run it normally with other unit tests but on demand ?

Reply:
I added a test category ("Integration") and moved all the integration tests to their own namespace. All we have is integration tests though, so better start adding unit tests.